src/main/kotlin/frc/robot/subsystems/gripper/GripperIOReal.kt

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.gripper
+
+import com.revrobotics.CANSparkBase
+import com.revrobotics.CANSparkLowLevel
+import com.revrobotics.CANSparkMax
+import com.revrobotics.SparkLimitSwitch
+import edu.wpi.first.units.Units
+import edu.wpi.first.wpilibj.DigitalInput
+import edu.wpi.first.wpilibj.RobotController
+import edu.wpi.first.wpilibj.Timer
+
+class GripperIOReal : GripperIO {
+    override val inputs = LoggedGripperInputs()
+    private val rollerMotor: CANSparkMax =
+        CANSparkMax(0, CANSparkLowLevel.MotorType.kBrushless) // TODO: Replace deviceID with real value
```

Comment:
You can determine this now.

Reply:
ed98a5be31f439d0ac3f590d617fae254c91d42a
eb8bc115e1da82b7347fbe2e539f56f2271cc27e